package.json

```diff
@@ -102,9 +104,18 @@
     "core-js": "^2.4.1",
     "express": "^4.15.3",
     "font-awesome": "^4.7.0",
+    "highlight.js": "^9.12.0",
     "jquery": "^3.1.1",
     "lc-form-validation": "^1.0.0",
-    "marksy": "^0.3.0",
+    "markdown-it": "^8.3.2",
+    "markdown-it-abbr": "^1.0.4",
```

Comment:
I don't know if all these libs are necessary, what do you think? @brauliodiez @JaimeSalas @crsanti @fjcalzado

Reply:
Yes guys, I know long dependency lists are ugly. `markdown-it` is the core library but the rest are just tiny features extensions installed as plugins. `markdown-it` encourages the use of already existing plugins instead of custom code, although this can also be done when no plugin is available.

Reply:
I would document them in a wiki and right now choose the ones we need. We can cherry pick others whenever we need them in next iterations.

Reply:
@fjcalzado can you add a page in our project wiki with a short list + explanation what is for each of them.

The ideas of emojis is something nice to add, people love that and is used to that

Reply:
Guys, I added the page to the wiki. Find it [here](https://github.com/Lemoncode/LeanMood/wiki/markdown-plugins).  Please take a look and let's decide which one you consider interesting for the first iteration and which one can be postponed to next releases.

Reply:
Just taking a look to the wiki, I like all the entries :D, we can let them as the user types, but right now not include them in the toolbar (we want to keep it simple right now).

Reply:
Sure! 👍  That was my initial idea, no extra buttons for the initial release, let's keep toolbar simple.

These could be 'advanced' features for now, only accessible from markdown syntax for the 'advanced' users. If any of them become popular, we can always add it to the toolbar in future iterations. 

All right.  One less issue 😅

Reply:
** DONE**